examples/linear_constant_advection.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import sys
+import math
+
+
+
+prepare_vars()
+
+
+N = 40
+I_Mesh['NODES_X'] = N
+I_Mesh['NODES_Y'] = N
+I_Mesh['NODES_Z'] = N
+
+I_Mesh['XMIN'] = -1.0
+I_Mesh['YMIN'] = -1.0
+I_Mesh['ZMIN'] = -1.0
+
+I_Mesh['XMAX'] = 1.0
+I_Mesh['YMAX'] = 1.0
+I_Mesh['ZMAX'] = 1.0
+
+I_TI['final_time'] = 1
+I_TI['cfl'] = 0.95
+
+dt = I_TI['cfl'] * 2.0 / float(N)
+num_steps = math.ceil(I_TI['final_time']/dt)
+dt = I_TI['final_time'] / num_steps
+
+I_TI['time_integrator'] = 'CarpenterKennedy2n54'
+
+I_TI['DT'] = dt
+I_TI['num_steps'] = num_steps
+
+# I_Tech['device'] = 1
+
+I_Tech['REAL'] = 'double'
+I_Tech['REAL4'] = I_Tech['REAL'] + "4"
```

Comment:
You might use `'4'` instead of `"4"` (since you are using mostly `'`). But this is a very minor comment.